### PR TITLE
Clearing up the docs on how to build from with the --localrpm option

### PR DESCRIPTION
**Pull request by fruch**
also added an option to download non scylla-core rpms,
so people won't need to built those every time

### CODE REVIEW

**Comment by fruch** [comment]
@eliransin after out talk today, I've update the docs here to try to make it a bit more clear

would love any input

**Comment by fruch** [comment]
@bentsi, when talked with @eliransin this week about this change.
he was concerned with those type of change, in regards that now developers won't know how to built the AMI.
I've update the docs a bit in this PR, to clearify. but you might want to mention it when you present the new AMI. and point people to here to try out the instructions.

File: aws/ami/build_ami.sh [hunk 1]
```diff
@@ -10,9 +10,12 @@ print_usage() {
     echo "  --repo-for-install  repository for install, specify .repo/.list file URL"
     echo "  --repo-for-update  repository for update, specify .repo/.list file URL"
     echo "  --product          scylla or scylla-enterprise"
+    echo "  --download-no-server  download all rpm needed but scylla from `repo-for-install`"
```
Comment (bentsi) [inline aws/ami/build_ami.sh]: except of Scylla

Comment (bentsi) [reply aws/ami/build_ami.sh]: excluding Scylla

Comment (fruch) [reply aws/ami/build_ami.sh]: fixed

**Pull request merged**